crates/l3/appchain-client/src/clients/mod.rs

```diff
@@ -17,6 +18,7 @@ impl<'a> StarknetCoreContractClient<'a> {
             operator: Operator::new(address, signer),
             messaging: Messaging::new(address, signer),
             core_contract: CoreContract::new(address, signer),
+            verifier_contract: VerifierContract::new(address, signer),
```

Comment:
the verifier is a separate contract altogether right? it's not really a part of the core contract.

Reply:
yes, restructured